browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -36,6 +42,19 @@ void BravePrivacyHandler::RegisterMessages() {
                                            base::Unretained(this)));
 }
 
+// static
+void BravePrivacyHandler::AddLoadTimeData(content::WebUIDataSource* data_source,
+                                          Profile* profile) {
+#if !BUILDFLAG(USE_GCM_FROM_PLATFORM)
+  gcm::BraveGCMChannelStatus* gcm_channel_status =
+      gcm::BraveGCMChannelStatus::GetForProfile(profile);
+
+  DCHECK(gcm_channel_status);
+  data_source->AddBoolean("pushMessagingEnabledAtStartup",
```

Comment:
should this just be `true` if `USE_GCM_FROM_PLATFORM`?

Reply:
done!